src/com/lftechnology/jan12/assignment/ListMaipulation.java

```diff
@@ -0,0 +1,41 @@
+package com.lftechnology.jan12.assignment;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * 1. Takes the words in its argument list and prints out any duplicate words, the number of distinct words, and a list of the words with
+ * duplicates eliminated. Hint: Try with HashSet and List implementation
+ * 
+ * This class is extract to get unique, distinct and duplicate words from the given {@link List}.
+ * 
+ * @author bibhushan<bibhushanjoshi@lftechnology.com>
+ * 
+ * 
+ * 
+ */
+public class ListMaipulation {
+	private static final Logger LOGGER = Logger.getLogger(ListMaipulation.class.getName());
+
+	public static void main(String[] args) {
+		List<String> nameList = new ArrayList<String>(Arrays.asList("a", "b", "c", "c", "d", "e", "b"));
+		ListUtility listChange = new ListUtility();
+		Set<String> uniques = new HashSet<String>();
```

Comment:
Better to give variable name like uniqueWords, duplicateWords, distinctWords, 
And these variable are plural because it is representing list of words